test/integrations/apache-custom-config.integration.test.ts

```
@@ -0,0 +1,211 @@
+        "файл.txt": "Unicode filename",
+        "中文文件.txt": "Chinese filename",
+      },
+      "deep": {
+        level: {
+          "deepFile.txt": "Deep nested file",
+        },
+      },
+    },
+    "many-files": Array.from({ length: 10 }, (_, i) => ({
+      [`file-${String(i + 1).padStart(3, "0")}.txt`]: `Content of file ${i + 1}`,
+    })).reduce((acc, obj) => ({ ...acc, ...obj }), {}),
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)